src/client/theme-default/index.ts

```diff
@@ -22,7 +23,10 @@ export { default as VPTeamMembers } from './components/VPTeamMembers.vue'
 
 const theme: Theme = {
   Layout,
-  NotFound
+  NotFound,
+  enhanceApp(ctx) {
+    ctx.app.component('Badge', VPBadge)
+  }
```

Comment:
The docs about how to extend default theme should also be updated.

Reply:
@meteorlxy I thought about it. But I wasn't sure because of the JS/TS difference.

In the documentations are an object given. Does this work like the same way or does the object in the docs represent the `ctx`? For this case it would be correct. Because `app` is part of the `ctx`.

That's the documentation I'm talking about: https://vitepress.vuejs.org/guide/using-vue#registering-global-components-in-the-theme

So I'm not sure if the following would be correct because of the brackets enclosing `ctx`.

```js
enhanceApp({ ctx }) {
  ctx.app.component('VueClickAwayExample', VueClickAwayExample)
}
```